wagtail/admin/forms/account.py

```diff
@@ -15,9 +16,14 @@
 from wagtail.admin.widgets import SwitchInput
 from wagtail.permissions import page_permission_policy
 from wagtail.users.models import UserProfile
+from wagtail.utils.utils import reduce_image_size
 
 User = get_user_model()
 
+WAGTAIL_USER_PROFILE_IMAGE_SIZE_BOUND = getattr(
+    settings, "WAGTAIL_USER_PROFILE_IMAGE_SIZE_BOUND", 1024
+)
```

Comment:
i contemplated if making the value arbitrary was the write choice, and thought to make it configurable for the user. But i guess you are right and wagtail setting a default might be a better option than having it configurable since developers might not necessarily want to be concerned with configuring it. Would make it default to 1mb.